src/main/java/io/anserini/ann/IndexVectors.java

```diff
@@ -65,6 +65,9 @@
     @Option(name = "-encoding", metaVar = "[word]", required = true, usage = "encoding must be one of {fw, lexlsh}")
     public String encoding;
```

Comment:
`fw` is better right? maybe set `fw` as default?

Reply:
sure, good point, I'll make `fw` default.